pom.xml

```diff
@@ -1757,6 +1757,13 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>26.15.0</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
This should still be kept at the `hudi-gcp` and `hudi-gcp-bundle` module.  My suggestion is to define `<gcp.libraries-bom.bersion>26.15.0</gcp.libraries-bom.bersion>` under `<properties>` in the root POM and use that to avoid hard coding the version.

Reply:
got it, made some changes,  PTAL, thanks!